docs/general-configuration.md

```diff
@@ -0,0 +1,49 @@
+---
+id: general-configuration
+title: General Configuration
+sidebar_label: General Configuration
+---
+
+## `General Description`
+
+General Configuration is the place where you can personalize your Plant an App application. 
+
+This can be done by entering `Configuration` from your application menu and then going onto the `General` page. After entering the `General` page, a page similar with the one below will appear:
```

Comment:
```suggestion
This can be done by entering `Configuration` from your application menu and then going into the `General` page. After entering the `General` page, a page similar to the one below will appear:
```